roles/satellite-clone/tasks/main.yml

```diff
@@ -1,10 +1,15 @@
 ---
-- include_vars: main.yml
-
 # include pre_install_check
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: "check that variables are updated"
+  fail: msg="Please update the variables in satellite-clone-vars.yml (after copying satellite-clone-vars.sample.yml)"
+  when:
+    - hostname == "satellite.example.com"
+    - activationkey == "changeme"
+    - org == "111111"
```

Comment:
I vote for the `or` operator here which will benefit the user more although the usage of list (for and operator) is cool :)